Automattic/jetpack
# Jetpack Sync: Custom Sync Queue Table: Add unit tests

## Proposed changes:

This PR is continuation of the spike #31318 and subsequent PRs: #32089, #32111, #32135 and #32275

It adds the unit test that should capture all the logic for the Custom Table feature.

It's a separate PR as we need to add exclusion for PHP5.6 for the tests, as they require PHP7+ to run.

More details to follow on this requirement.

### Other information:


## Jetpack product discussion

p9dueE-7e6-p2

## Does this pull request change what data or activity we track or use?

No

## Testing instructions:

* Apply PR
* Make sure Jetpack Sync works
*  Run the Docker PHPUnit tests - jetpack docker phpunit and make sure they pass. You can also run only the Sync unit tests with jetpack docker phpunit -- --testsuite "sync"
* All the steps in the testing instructions comment in p9dueE-7e6-p2
